doc/source/reference/commands/tokenize.rst

```diff
@@ -212,6 +213,37 @@ See :doc:`/reference/token_filters` about token filters.
 
 .. _tokenize-return-value:
```

Comment:
I will fix the anchor? now.

Reply:
fix: [9d415a3](https://github.com/groonga/groonga/pull/2311/commits/9d415a3729a15e2abd2be3d0d09c9b193f18b691)